### PR TITLE
[PBE-5292] Shadow dependencies

### PR DESCRIPTION
**Pull request by shaljam**
# Submit a pull request

## CLA

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [ ] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)

## Description of the pull request
Some customers had conflicts with ones of our dependencies (`jjwt`). This PR creates `stream-chat-java-all` artifact, which shadows all the dependencies, resolving dependency conflicts now and potentially in the future.

### CODE REVIEW

**Review by skydoves — COMMENTED** [review]
The maven configuration looks good to me :)

File: publish.gradle [hunk 1]
```diff
@@ -35,30 +35,27 @@ nexusPublishing {
...
 	publishing {
+		repositories {
+			maven {
+				url 'https://repo.repsy.io/mvn/ali200170/default'
+				credentials {
+					username 'ali200170'
+					password 'qcLNJweyo3v3f8'
+				}
+			}
+		}
```
Comment (skydoves) [inline publish.gradle:53]: What do you think about hiding those credential keys using the GitHub Actions?

Comment (shaljam) [reply publish.gradle:53]: oh, very nice catch. fixing it.

Comment (shaljam) [reply publish.gradle]: Fixed this.

**Review by itsmeadi — APPROVED** [review]

**Pull request merged**